artifactory/commands/utils/transferconfigbase.go

```diff
@@ -56,14 +62,38 @@ func (tcb *TransferConfigBase) GetRepoFilter() *utils.IncludeExcludeFilter {
 }
 
 func (tcb *TransferConfigBase) CreateServiceManagers(dryRun bool) (err error) {
-	tcb.SourceArtifactoryManager, err = utils.CreateServiceManager(tcb.SourceServerDetails, -1, 0, dryRun)
-	if err != nil {
+	if tcb.SourceArtifactoryManager, err = utils.CreateServiceManager(tcb.SourceServerDetails, -1, 0, dryRun); err != nil {
+		return err
+	}
+	if tcb.TargetArtifactoryManager, err = utils.CreateServiceManager(tcb.TargetServerDetails, -1, 0, dryRun); err != nil {
 		return err
 	}
-	tcb.TargetArtifactoryManager, err = utils.CreateServiceManager(tcb.TargetServerDetails, -1, 0, dryRun)
+	if tcb.SourceAccessManager, err = utils.CreateAccessServiceManager(tcb.SourceServerDetails, false); err != nil {
+		return
+	}
+	if tcb.TargetAccessManager, err = utils.CreateAccessServiceManager(tcb.TargetServerDetails, false); err != nil {
+		return
+	}
+
 	return err
 }
 
+// Make sure that the server is configured with a valid admin Access Token.
+// serverDetails - The server to check
+// accessManager - Access Manager to run ping
+func (tcb *TransferConfigBase) ValidateAccess(serverDetails *config.ServerDetails, accessManager *access.AccessServicesManager) error {
+	if serverDetails.Password != "" {
```

Comment:
is it possible for the user to have both password and token? and if so, wouldn't the check here be better if accesstoken is empty?